src/nvim/ex_docmd.c

```diff
@@ -2900,7 +2900,7 @@ const char * set_one_cmd_context(
     p = (const char *)xp->xp_pattern;
     while (*p != NUL) {
       if (has_mbyte) {
-        c = mb_ptr2char((const char_u *)p);
+        c = utf_ptr2char((const char_u *)p);
```

Comment:
If you are keeping only the branch where `has_mbyte` is true, can you be consistent with the rest?

Reply:
Good catch, thanks. Will fix all these cases you pointed out.